server/controllers/finance/debtors/index.js

```diff
@@ -36,8 +36,8 @@ function list(req, res, next) {
   const sql = `
     SELECT BUID(d.uuid) AS uuid, BUID(d.group_uuid) AS group_uuid,
       d.text, map.text as hr_entity
-    FROM debtor d
-    JOIN entity_map map ON map.uuid = d.uuid;
+    FROM debtor d 
```

Comment:
If some debtors are missing an entity map, than that is a bug.  A trigger should create an an entity_map record anytime a debtor-type entity is created.

Do you have a reproducible test case?